<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1021</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>473</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>310</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>491</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/410.bmp</normaloff>:/images/images/410.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>QListWidget操作</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QToolButton" name="tBtnListAppend">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>tBtnListAppend</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="tBtnListInsert">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>tBtnListInsert</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QToolButton" name="tBtnListDelete">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>tBtnListDelete</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="tBtnListIni">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>tBtnListIni</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="tBtnListClear">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>tBtnListClear</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>491</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/408.bmp</normaloff>:/images/images/408.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>QTreeWidget操作</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>491</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/images/412.bmp</normaloff>:/images/images/412.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>QTableWidget操作</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>QListWidget</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>当前项变化</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkBoxListEditable">
            <property name="text">
             <string>可编辑</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="tBtnSelectItem">
            <property name="text">
             <string>tBtnSelectItem</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::MenuButtonPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnSelALL">
            <property name="text">
             <string>tBtnSelALL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnSelNone">
            <property name="text">
             <string>tBtnSelNone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tBtnSelInvs">
            <property name="text">
             <string>tBtnSelInvs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <item>
          <property name="text">
           <string>新建项目</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/images/images/check2.ico</normaloff>:/images/images/check2.ico</iconset>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建项目2</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/images/check2.ico</normalon>
           </iconset>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>QTreeWidget</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>QTableWidget</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListIni"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actListIni">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/129.bmp</normaloff>:/images/images/129.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/delete1.bmp</normaloff>:/images/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/424.bmp</normaloff>:/images/images/424.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/316.bmp</normaloff>:/images/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/324.bmp</normaloff>:/images/images/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actSelALL">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit2.bmp</normaloff>:/images/images/exit2.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/124.bmp</normaloff>:/images/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actSelPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actSelInvs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
